_includes/configuration/inline.md

```diff
@@ -23,6 +21,28 @@ tinymce.init({
   inline: true
 });
 ```
+
 > **Note**: {{site.productname}} in inline mode will not initialize on the following elements: `area`, `base`, `basefont`, `br`, `col`, `frame`, `hr`, `img`, `input`, `isindex`, `link`, `meta`, `param`, `embed`, `source`, `wbr`, `track`, `colgroup`, `option`, `table`, `tbody`, `tfoot`, `thead`, `tr`, `th`, `td`, `script`, `noscript`, `style`, `textarea`, `video`, `audio`, `iframe`, `object`, and `menu`.
 
 For more information on the differences between the editing modes, please see this page [here]({{ site.baseurl }}/general-configuration-guide/use-tinymce-inline/).
+
+
+### Inline editing and complex stylesheets
```

Comment:
this isn't necessarily "complex" stylesheets - even your example is very simple. it's more about... interaction between the inline editor/its content and stylesheets?

Reply:
Agreed, Maybe
> CSS stylesheets and inline editor content